cubeb-backend/src/capi.rs

```diff
@@ -49,6 +49,7 @@ macro_rules! capi_new(
                 Some($crate::capi::capi_stream_reset_default_device::<$stm>),
             stream_get_position: Some($crate::capi::capi_stream_get_position::<$stm>),
             stream_get_latency: Some($crate::capi::capi_stream_get_latency::<$stm>),
+            stream_get_input_latency: Some($crate::capi::capi_stream_get_input_latency::<$stm>),
```

Comment:
With `stream_get_input_latency` introduced, we will have two similar function names
- `stream_get_input_latency`
- `stream_get_latency`

How about renaming `stream_get_latency` to `stream_get_output_latency`? It definitely looks more understandable to me.

Reply:
It's unnecessary ABI breakage, but yeah.